csm/conf/csm_cleanup.py

```diff
@@ -32,6 +32,13 @@
 import os
 import pathlib
 
+index_field_map = {
```

Comment:
Add spacing between ':' and ',' wherever required.

Reply:
Comment addressed. Please review